### PR TITLE
Szintefestes

### PR DESCRIPTION
**Pull request by DannySS123**

### CODE REVIEW

File: src/content/posts/2022-09-30-szintfestes.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+A szintfestés a Schönherz közéletének egy nagyon fontos része: ez egy olyan alkalom, ahol a kollégium lakói mindannyian festhetnek a szintjük falára logókat vagy bármilyen vicces, funky ábrákat. (Bizonyos határokon belül.)
+
+Ezek az alkotások generációkon át díszítik a Schönherz belsejét, amíg újra nem festik az egész szintet, helyet adva újabb lehetőségeknek.
+
+## 2022 őszi szintfestés
+
+![szintfestes-group-pic](../images/posts/kirdev_szintfestes.png)
+
+Idén a szintfestés alól a Schönherz 13. emelete, a Simonyi szakkolléium szintje se maradhatott ki. Még egészen sok üres hely volt a falakon, így már igazán itt volt az ideje, hogy színesebbé tegyük a folyosókat.
```
Comment (triszt4n) [inline src/content/posts/2022-09-30-szintfestes.md:25]: * Simonyi szakkolléium -> Simonyi Károly Szakkollégium

File: src/content/posts/2022-09-30-szintfestes.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+A Kir-Dev csapata is részt vett ebben. Még a szintfestés napja előtt megterveztünk és felrajzoltuk a kör logójának körvonalát, majd a várva várt napont egy kellemes délutánt töltöttek tagjaink a festegetéssel. Ugyan nem a festés az erősségünk, mégis sikerült egy csodálatos logót alkotnunk, ráadásul jövőre is vannak már elképzeléseink, hogy a logó köré miket szeretnénk festeni.
```
Comment (triszt4n) [inline src/content/posts/2022-09-30-szintfestes.md:27]: * napont -> napon

File: src/content/posts/2022-09-30-szintfestes.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Eddig még soha nem volt Kir-Dev logó festeve a szakkoli szintjén, úgyhogy már nagyon itt volt az ideje.
```
Comment (triszt4n) [inline src/content/posts/2022-09-30-szintfestes.md:28]: * festeve -> festve

File: src/content/posts/2022-09-30-szintfestes.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+![kirdev-logo-phases](https://warp.sch.bme.hu/images/kirdev-logo-phases)
+
+Viszont a tagjaink tevékenysége a logónk befejeztével se maradt abba, úgyanis egy másik logó is nagyon hiányzott már a szintről, mégpedig a szokkollégium logója. A Simonyi sok köréből érkeztek lekes emberek, és közös erővel megalkottuk a gyönyörű Simonyi logót is. Ha valaki a 13. szintre érkezik, most már egyből tudni fogja, hogy jó helyen jár.
```
Comment (triszt4n) [inline src/content/posts/2022-09-30-szintfestes.md:32]: * úgyanis -> ugyanis
* szokkollégium -> szakkollégium
* lekes -> lelkes

**Review by triszt4n — CHANGES_REQUESTED** [review]
Nice stuff, see my typo requests

**Review by Tschonti — APPROVED** [review]

**Review by SepsiLaszlo — APPROVED** [review]
Great work, I feel really inspired after reading it! :rocket:

**Review by triszt4n — APPROVED** [review]
Let's brag about our community 👑

**Pull request merged**